src/reanimated2/js-reanimated/JSReanimated.ts

```diff
@@ -71,4 +115,38 @@ export default class JSReanimated extends NativeReanimated {
   unsubscribeFromKeyboardEvents(_: number): void {
     // noop
   }
+
+  initializeSensor(sensorType: SensorType, interval: number): WebSensor {
+    const config =
+      interval === -1
+        ? { referenceFrame: 'device' }
+        : { frequency: 1 / interval };
```

Comment:
I think it should be`{ frequency: 1000 / interval }`, because the `interval` contains milliseconds.